pd/src/app.rs

```diff
@@ -39,6 +40,9 @@ const NUM_RECENT_ANCHORS: usize = 64;
 /// The Penumbra ABCI application.
 #[derive(Debug)]
 pub struct App {
+    /// Parameters for the chain (determined at genesis and thenceforth by governance).
+    pub chain_params: ChainParams,
```

Comment:
Should this be `pub` ?  Thoughts:

(1) Currently we don't ever access the `App` externally, we wrap it in a bunch of Tower stuff, so I'm not sure what would be accessing it;
(2) IME it's useful to have POD structures with public fields, or structures with invariants and private fields, but I haven't had great success with mixing the two.

Reply:
Makes sense to make it private to me -- done.